test/ctia/http/routes/observable_test.clj

```diff
@@ -1,100 +1,312 @@
 (ns ctia.http.routes.observable-test
```

Comment:
There is value in readable examples in tests.  Even though they are long-winded, the artifact of the test is documentation, almost like a CURL example (but way better IMO).  *Maybe* generative testing will be able to replace it, but not yet.  For one thing, everyone on the team will need to be good at writing and debugging generative tests.  We're not there yet.  In the mean time, we need to keep scenario oriented, hand-written tests maintained while also maintaining separate generative tests when they augment the hand-written tests.

I *feel* like generative tests will usually give the most value for referentially-transparent code that doesn't state management.  That said, I do feel like the generative tests that we have written for the HTTP APIs were helpful.  (They were probably helpful mostly because our test coverage was poor).  IMO the jury is still out.

I've restored the original observable tests.  They may be ugly, but I actually feel like they are examples of very good testing (it tests a real scenario that a user might have to to follow using the API).